test.sh

```diff
@@ -59,6 +59,13 @@ function usage() {
    echo "To run CI-regression tests       : INCLUDE_SLOW_TESTS=true ./${Me}"
    echo "To run nightly regression tests  : RUN_NIGHTLY_TESTS=true ./${Me}"
    echo "To run make build-and-test tests : RUN_MAKE_TESTS=true ./${Me}"
+   echo
+   echo "To run a smaller collection of slow running tests,
+name the function that drives the test execution.
+Examples:"
+   echo "  INCLUDE_SLOW_TESTS=true ./test.sh run_btree_tests"
+   echo "  INCLUDE_SLOW_TESTS=true ./test.sh run_splinter_functionality_tests"
+   echo "  INCLUDE_SLOW_TESTS=true ./test.sh nightly_cache_perf_tests"
```

Comment:
Slightly unrelated change. I needed a way to quickly re-test batches of tests run by this script under `ASAN` build modes. So, added this new capability to run a set of tests w/o having to wait for full runs to complete to validate a fix.